packages/core/include/soss/Instance.hpp

```diff
@@ -25,11 +25,12 @@
 #include <string>
 #include <unordered_map>
 #include <vector>
+#include "soss/soss_export.hpp"
```

Comment:
Super nitpick: I recommend only using the quote `"` version of include when specifying paths that are relative to the current header file location, since that's [the key difference between `"` and `<`](https://stackoverflow.com/a/21594).

That would also be more consistent with the other include directives in the project.